external/mmsegmentation/segmentation_tasks/apis/segmentation/openvino_task.py

```diff
@@ -164,25 +164,36 @@ def infer(self,
               inference_parameters: Optional[InferenceParameters] = None) -> DatasetEntity:
         if inference_parameters is not None:
             update_progress_callback = inference_parameters.update_progress
-            dump_saliency_map = not inference_parameters.is_evaluation
+            dump_soft_prediction = not inference_parameters.is_evaluation
         else:
             update_progress_callback = default_progress_callback
-            dump_saliency_map = True
+            dump_soft_prediction = True
 
         dataset_size = len(dataset)
         for i, dataset_item in enumerate(dataset, 1):
-            predicted_scene, feature_vector, saliency_map = self.inferencer.predict(dataset_item.numpy)
+            predicted_scene, feature_vector, soft_prediction = self.inferencer.predict(dataset_item.numpy)
             dataset_item.append_annotations(predicted_scene.annotations)
 
             if feature_vector is not None:
                 feature_vector_media = TensorEntity(name="representation_vector", numpy=feature_vector.reshape(-1))
                 dataset_item.append_metadata_item(feature_vector_media, model=self.model)
 
-            if dump_saliency_map and saliency_map is not None:
-                saliency_map_media = ResultMediaEntity(name="saliency_map", type="Saliency map",
-                                                annotation_scene=dataset_item.annotation_scene, 
-                                                numpy=saliency_map, roi=dataset_item.roi)
-                dataset_item.append_metadata_item(saliency_map_media, model=self.model)
+            if dump_soft_prediction:
+                for label_index, label in self._label_dictionary.items():
+                    if label_index == 0:
+                        continue
+                    if len(soft_prediction.shape) == 3:
+                        current_label_soft_prediction = soft_prediction[:, :, label_index]
+                    else:
+                        current_label_soft_prediction = soft_prediction
+                    class_act_map = get_activation_map(current_label_soft_prediction)
+                    result_media = ResultMediaEntity(name=f'{label.name}',
+                                                     type='Soft Prediction',
+                                                     label=label,
```

Comment:
'name' & 'type' convention is inconsistent compared to mmseg pytorch task.

Reply:
fixed